mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/CassandraMailboxManagerProvider.java

```diff
@@ -114,7 +114,7 @@ private static CassandraMailboxManager provideMailboxManager(CqlSession session,
             LIMIT_ANNOTATIONS, LIMIT_ANNOTATION_SIZE);
 
         SessionProviderImpl sessionProvider = new SessionProviderImpl(noAuthenticator, noAuthorizator);
-        CassandraPerUserMaxQuotaManager maxQuotaManager = new CassandraPerUserMaxQuotaManager(new CassandraPerUserMaxQuotaDao(session),
+        CassandraPerUserMaxQuotaManagerV1 maxQuotaManager = new CassandraPerUserMaxQuotaManagerV1(new CassandraPerUserMaxQuotaDao(session),
```

Comment:
Shouldn't we use V2 for tests if this is deprecated? (it's nice I see I missed that in my own work as well)